wasmplugin/rules/coraza-demo.conf

```diff
@@ -222,7 +222,8 @@ SecDebugLogLevel 3
 # level response status codes).
 #
 SecAuditEngine RelevantOnly
-SecAuditLogRelevantStatus "^(?:(5|4)(0|1)[0-9])$"
+# SecAuditLogRelevantStatus includes status 0 as a workaround for https://github.com/corazawaf/coraza/pull/1097
+SecAuditLogRelevantStatus "^(?:(5|4)(0|1)[0-9]|0)$"
```

Comment:
Considering that we are enabling support for audit logs, I propose to ship also this workaround in order to have an expected experience with default CRS interruptions treated as relevant for audit log purposes.
More context: https://github.com/corazawaf/coraza/pull/1097
WDYT @jcchavezs? 
Once a patch/minor release of Coraza is out, we will be able to revert this workaround.

Reply:
Reverted, fix is out in Coraza master, pointing directly to it